client/src/components/Workflow/Editor/Index.vue

```diff
@@ -422,6 +423,9 @@ export default {
             this.$refs.lint.refresh();
             showLint();
         },
+        onUpgrade() {
+            this.attemptRefactor([{'action_type': 'upgrade_workflow'}]);
```

Comment:
One small problem is that the change isn't visible immediately, the page has to be refreshed. I tried a few things without success, hopefully this is fixable in about 5 seconds for someone who actually knows JS. :)